includes/bsf-rt-user-manual.php

```diff
@@ -13,12 +13,12 @@
 <div class="bsf_rt_user_manual">
 	<br><label class="bsf_rt_page_title" for="howtouse">
 		<?php
-		echo 'How to Use? </label><br><br>
+		echo __( 'How to Use? </label><br><br>
```

Comment:
We can separate out each and every statement in translation and keep the HTML out of the PHP script.
Step 1,2,3 in a single `__()` and rest of the statement in other `__()`
```php
<b><?php __( 'Step 1' ); ?></b> <?php __( ':Under the <i>General Settings</i> Tab, select post types to display the Read Time and Progress bar. Select Words Per Minute and set other options if required.' ) ?> <br><br>
```